### ISSUE TITLE
Bug(scope): TranslocoService.Translate and Translate have different generic type signatures

### ISSUE THREAD

**Issue by seadonk** [ngneat/transloco#544]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Transloco package(s) are the source of the bug?

Transloco

### Is this a regression?

No

### Current behavior

The generic type on the standalone translate function does not have a default, and thus defaults to `unknown`. 
The TranslocoService.translate method is typed with a default of `string`.

This is breaking compilation in some cases that use the standalone function because `Argument of type 'unknown' is not assignable to parameter of type 'string'`.

### Expected behavior

The standalone translate function, and the TranslocoService.translate method should both have a generic type default of `string`.

### Please provide a link to a minimal reproduction of the bug

-

### Please provide the environment you discovered this bug in

```markdown
Transloco: 3.1.3
Angular: 13.1.3
Node: 16.13.1
Package Manager: Yarn
OS: Windows
```

### I would like to make a pull request for this bug

Yes 🚀

**Comment by seadonk** [comment]
Created [PR #545](https://github.com/ngneat/transloco/pull/545)

**Comment by shaharkazaz** [comment]
Fixed in v3.1.4 👌 
Thank you Brain @seadonk!
